pkg/build/builder/docker.go

```diff
@@ -147,12 +150,20 @@ func (d *DockerBuilder) Build() error {
 
 	if push && pushTag != "" {
 		// Get the Docker push authentication
-		pushAuthConfig, authPresent := dockercfg.NewHelper().GetDockerAuth(
-			pushTag,
+		pushAuthConfig, authPresent := dockercfg.NewHelper().GetAllDockerAuths(
```

Comment:
Probably don't need `authPresent` here if below you check if the push config exists for the _particular_ registry.

Reply:
agreed .... and of course, whether you have auth for the actual registry is not as critical in the mirror scenario, as long as you have the auth for the mirror